packages/ui-extensions/docs/surfaces/point-of-sale/reference/examples/targets/pos-draft-order-details-action.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  Navigator,
+  Screen,
+  ScrollView,
+  Text,
+  extension,
+} from '@shopify/ui-extensions/point-of-sale';
+
+export default extension(
+  'pos.draft-order-details.action.render',
+  (root, api) => {
+    const navigator = root.createComponent(Navigator);
+    const screen = root.createComponent(Screen, {
+      name: 'DraftOrderDetailsAction',
+      title: 'Draft Order Details Action',
+    });
+    const scrollView = root.createComponent(ScrollView);
+    const text = root.createComponent(Text);
+
+    text.append(`ID for current draft order screen: ${api.order.id}`);
```

Comment:
```suggestion
    text.append(`ID for current draft order screen: ${api.draftOrder.id}`);
```

Same with the other examples